rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -49,13 +82,60 @@ def test_hop_l2_deposit(database, ethereum_inquirer):
             asset=A_ETH,
             balance=Balance(amount=FVal('0.2')),
             location_label=ADDY,
-            notes='Bridge 0.2 ETH to Optimism at the same address via Hop protocol',
+            notes='Bridge 0.2 ETH to Optimism via Hop protocol',
             counterparty=CPT_HOP,
             address=string_to_evm_address('0xb8901acB165ed027E32754E0FFe830802919727f'),
+            extra_data={'sent_amount': '0.2'},
         )]
     assert expected_events == events
 
 
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xD8245043a3f339400dcfFFc7A9E6F22a264b933D']])
+def test_hop_l2_deposit_usdc(database, ethereum_inquirer, ethereum_accounts):
+    """Data taken from
+    https://etherscan.io/tx/0xd46640417a686b399b2f2a920b0c58a35095759365cbe7b795bddec34b8c5eee
+    """
+    tx_hash = deserialize_evm_tx_hash('0xac42ca2d88194c0ee219f6c71c98fe566667151a1dc235d17d993b6985342062')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp = TimestampMS(1710955643000)
+    gas_fee, bridge_amount = '0.005802319111323689', '3700'
```

Comment:
In this and other tests putthem in one line

```suggestion
    timestamp, gas_fee, bridge_amount = TimestampMS(1710955643000), '0.005802319111323689', '3700'
```